draft-ietf-ohai-ohttp.md

```diff
@@ -429,9 +429,21 @@ Change controller:
 
 # HPKE Encapsulation
 
-HTTP message encapsulation uses HPKE for request and response encryption.  An
-encapsulated HTTP message includes a binary-encoded HTTP message and no other
-content; see {{BINARY}}.
+HTTP message encapsulation uses HPKE for request and response encryption.
+
+An encapsulated HTTP request includes a timestamp and a binary-encoded HTTP
+message {{BINARY}}; see {{fig-req-pt}}.
+
+~~~
+Request {
+  Timestamp (32),
```

Comment:
32 bits of seconds seems quite wide, though it is the smallest reasonable power of 2 that buys us at least a day's worth of clock skew. In a perfect world, I'd prefer this be 16 bits or shorter. Do we have up-to-date data on clock skew we can use to make a more informed choice here?

Reply:
cc @tfpauly and @davidben, who might have a better view from the client